### DISCUSSION TITLE
looks_like field requirement

### DISCUSSION THREAD

**Discussion by secretstamos** [I-am-Erk/CDDA-Tilesets#1621] [General]
I believe there should be a mandate for looks_like for all new content (items/monster/terrain/furniture) in the main repo. 

It takes significantly less development time for the developer adding the item/monster/ect to fill out this field than it does for every tileset to both identify the missing icon and add it. 

At least in my own experience this is the largest loss of efficiency.

**Comment by Fris0uman** [comment]
Would probably be more efficient to open an issue about it in the main repo

**Reply by secretstamos** [reply]
Got it, will do

**Reply by secretstamos** [reply]
https://github.com/CleverRaven/Cataclysm-DDA/issues/61285
